tests/python_package_test/test_dask.py

```diff
@@ -0,0 +1,211 @@
+import os
+import sys
+
+import pytest
+if not sys.platform.startswith("linux"):
+    pytest.skip("lightgbm.dask is currently supported in Linux environments", allow_module_level=True)
+
+import dask.array as da
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+import scipy.sparse
+from dask.array.utils import assert_eq
+from dask_ml.metrics import accuracy_score, r2_score
+from distributed.utils_test import client, cluster_fixture, gen_cluster, loop  # noqa
+from sklearn.datasets import make_blobs, make_regression
+
+import lightgbm
+import lightgbm.dask as dlgbm
+
+data_output = ['array', 'scipy_csr_matrix', 'dataframe']
+data_centers = [[[-4, -4], [4, 4]], [[-4, -4], [4, 4], [-4, 4]]]
+
+pytestmark = [
+    pytest.mark.skipif(os.getenv("TASK", "") == "mpi", reason="Fails to run with MPI interface")
```

Comment:
Any ideas why it happens?

Reply:
Does it fail on every MPI job or only on `MPI source` and **NOT** on `MPI pip`?

Reply:
I think we can resolve this after this PR